### DISCUSSION TITLE
Prevent the screen from locking

### DISCUSSION THREAD

**Discussion by Nosnahc** [Docile-Alligator/Infinity-For-Reddit#1349] [Ideas]
Hello,
I find it quite disturbing that my screen locks while I'm watching a video.
Maybe a feature similar to what we could find on Newpipe would be interesting?

**Comment by Wladefant** [comment]
Can you tell more about the newpipe feature?

**Reply by Nosnahc** [reply]
Currently, I set my screen auto-lock after 30 seconds of inactivity. When I watch a video on youtube (newpipe in this case) the application keeps my screen on the time of the video Other apps do the same. However on Infinity, when I watch a video (or take a little time to read a comment without touching the screen) my screen locks itself. I think a feature that keeps the screen on when you're in Infinity (or at least when watching a video) would be very nice.

**Comment by Wladefant** [comment]
Thank you, could you provide a short screenrecording?

**Comment by Nosnahc** [comment]
I intentionally reduced the lock time to 15 seconds to make the video shorter.
[Video here](https://imgur.com/a/auflXB3)
